test/apps/index.js

```diff
@@ -0,0 +1,192 @@
+'use strict';
+
+const assert = require('assert');
+
+describe('apps', function () {
+	const libApp = require('lib/app');
+
+	// allow lib/app configuration override
+
+	let currentAppsConfig;
+	let currentCommandCenterSetup;
+
+	const oldFns = {
+		getAppsConfig: libApp.getAppsConfig,
+		getLogger: libApp.getLogger,
+		createCommandCenter: libApp.createCommandCenter
+	};
+
+	const testFns = {
+		getAppsConfig: function () {
+			return currentAppsConfig;
+		},
+		getLogger: function () {
+			return {
+				debug: function () {}
+			};
+		},
+		createCommandCenter: function (app) {
+			return {
+				setup: function () {
+					currentCommandCenterSetup(app);
+				}
+			};
+		}
+	};
+
+	beforeEach(function () {
+		libApp.getAppsConfig = testFns.getAppsConfig;
```

Comment:
I'll have a look, thanks 👍 
My general attitude is one where I'm trying as much as possible to keep things simple (that is: you read the code and you know immediately what it does, because there's ideally no library in between). But at some point, a library becomes useful enough to convince me otherwise ;)